packages/ember-input-validation/src/components/prefabs/tpk-validation-currency.gts

```diff
@@ -15,35 +27,34 @@ export interface TpkValidationCurrencyPrefabSignature
   Element: HTMLDivElement;
 }
 
-
 export default class TpkValidationCurrencyPrefabComponent extends Component<TpkValidationCurrencyPrefabSignature> {
   get mask() {
```

Comment:
tu peux stocker le masque en dehors, vu que il est statique pas besoin de réinstancier un objet à chaque fois

Reply:
Alors il dépend tout de même d'un argument (this.args.scale), je le stocke en dehors tout de même ?